<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgSerial</class>
 <widget class="QDialog" name="dlgSerial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,10,1">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>PortName</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbxPortName"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Baudrate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numudBaudrate">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>100000000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Parity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbxParity">
       <property name="currentIndex">
        <number>4</number>
       </property>
       <item>
        <property name="text">
         <string>Odd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Even</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SpaceParity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MarkParity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NoParity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UnknownParity</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>StopBit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbxStopBit">
       <item>
        <property name="text">
         <string>One</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>One And Half</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Two</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unknown</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>DataBits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbxDataBits">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>Data 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Data 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Data 7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Data 8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unknown</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgSerial</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgSerial</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
